packages/example-site/README.md

```diff
@@ -1,16 +1,44 @@
-# Example site with defaults
+# Example Site with example content
+
+https://gatsby-theme-project-portal-ex-site.netlify.app/
 
 A usage of
 [gatsby-theme-project-portal](https://github.com/thepolicylab-projectportals/project-portal-theme)
-that loads the theme and shows the default set of features without changing anything. 
+that serves as a local development environment designed to facilitate the testing and development of new features for the Project Portal theme. 
+
+
+
+## Quick Start Guide
+
+Once the repository is cloned, 
```

Comment:
I'd consider turning this into an imperative sentence, and add a period rather than a comma at the end.
```suggestion
Clone the repository. 
```
----
Another alternative is to make the whole thing into a series of bullets which can be read as a grammatically correct paragraph. (I prefer the way you have it now and would just make the grammatical correction above, but I'll include this for completeness.)

- Once the repository is cloned,
- Make sure you are at the root of the repository and run the following commands: 
  - to install the required dependencies:
    ```shell
    yarn install
    ```
  - to generate a built version of your website:
    ```shell
    yarn workspace example-site build
    ```
  - ... and then preview the production build locally:
    ```shell
    yarn workspace example-site serve
    ```